src/data/pastProjects.ts

```diff
@@ -0,0 +1,179 @@
+export const pastProjects = [
+    {
+        title: "nuCloud",
+        description: "The 2024 team developed a terminal deoxynucleotidyl transferase (TdT)-based solid phase DNA synthesis platform that allows users to synthesize data-encoding DNA.",
+        icon: "../../../static/past-project-icons/2024_nuCloud.gif",
```

Comment:
probably these filepaths should just have the file name, and then the `../../../static/past-project-icons/` part can get automatically prepended inside the actual `ProjectBlock` component